[Bug] Fix test failures

@google-jules

The test case, test_consultant_response_schema, is now failed on current code base.
Please help to fix it.


=================================== FAILURES ===================================
_______________________ test_consultant_response_schema ________________________

    def test_consultant_response_schema():
        """Test A (Schema Compliance): The output MUST validate against the JSON Schema."""
        valid_data = {
            "diagnosis_id": "RCA-BSP-001",
            "confidence": 0.9,
            "status": "CRITICAL",
            "root_cause_summary": "Null Pointer Dereference",
            "evidence": ["Timestamp 1456.78: null pointer dereference"],
            "sop_steps": [
                {
                    "step_id": 1,
                    "action_type": "CODE_PATCH",
                    "instruction": "Add check for clk_ptr before access.",
                    "expected_value": "N/A",
                    "file_path": "drivers/gpu/drm/msm/mdss.c"
                }
            ]
        }
>       response = ConsultantResponse(**valid_data)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for ConsultantResponse
E       confidence_score
E         Field required [type=missing, input_value={'diagnosis_id': 'RCA-BSP...s/gpu/drm/msm/mdss.c'}]}, input_type=dict]
E           For further information visit https://errors.pydantic.dev/latest/v/missing

tests/test_supervisor.py:25: ValidationError
=============================== warnings summary ===============================

--- a/tests/test_supervisor.py
+++ b/tests/test_supervisor.py
@@ -8,7 +8,7 @@ def test_consultant_response_schema():
     """Test A (Schema Compliance): The output MUST validate against the JSON Schema."""
     valid_data = {
         "diagnosis_id": "RCA-BSP-001",
-        "confidence": 0.9,
+        "confidence_score": 0.9,
         "status": "CRITICAL",
         "root_cause_summary": "Null Pointer Dereference",
         "evidence": ["Timestamp 1456.78: null pointer dereference"],

